android/jni/examples.mk

```diff
@@ -18,7 +18,7 @@
 
 LOCAL_PATH:= $(call my-dir)
 LIBUSB_ROOT_REL:= ../..
-LIBUSB_ROOT_ABS:= $(LOCAL_PATH)/../..
```

Comment:
Should this just be `$(LOCAL_PATH)/..`? But then `LIBUSB_ROOT_REL` would just be `..`

Reply:
fixed